MoreLinq/Generate.cs

```diff
@@ -36,7 +36,7 @@ static partial class MoreEnumerable
         /// </remarks>
         /// <example>
         /// <code>
-        /// IEnumerable&lt;int&gt; result = Sequence.Generate(2, n => n * n).Take(5);
+        /// var result = MoreEnumerable.Generate(2, n => n * n).Take(5);
```

Comment:
I fixed the name typo (`Sequence` &rarr; `MoreEnumerable`) in b43afc13f588f41dfb9e30182605a9fbbc75bbe2 as it's unrelated to this PR.